### PR TITLE
Nvlag

### PR DESCRIPTION
**Pull request by morgoth6**
Definitely not something to merge, but maybe something to test. The NVIDIA lag's are sometimes insane and using 4k display even more :/

Quick and dumb test to mitigate by switching glFinish()->glFlush() and atempt to use has_GLX_EXT_swap_control_tear. To be honest not sure how much it helps (or not) as I become so used to it that this can be placebo efect.

### CODE REVIEW

File: src/render.c [hunk 1]
```diff
@@ -968,7 +968,7 @@ win_blur_background(session_t *ps, struct managed_win *w, xcb_render_picture_t t
 /// region = ??
 /// region_real = the damage region
 void paint_all(session_t *ps, struct managed_win *t, bool ignore_damage) {
-	if (ps->o.xrender_sync_fence || (ps->drivers & DRIVER_NVIDIA)) {
```
Comment (sandsmark) [inline src/render.c:971]: this makes it worse here

**Comment by sandsmark** [comment]
in case you didn't see it: https://git.sailfishos.org/mer-core/libsdl/commit/771a34fc63f9dd6e2bbe65d17c18882e5c87a6be?view=parallel&w=1

**Comment by morgoth6** [comment]
> in case you didn't see it: https://git.sailfishos.org/mer-core/libsdl/commit/771a34fc63f9dd6e2bbe65d17c18882e5c87a6be?view=parallel&w=1

No. Looking at version of NVIDIA drivers mentioned there I guess problem is already fixed. Anyway see no problem to try it. Pushed something looking more or less as it and removed forced xrender fence forgotten in last batch.

**Comment by LoganDark** [comment]
I don't think the lag can be avoided. Even on Intel integrated graphics, on a 4k screen, audio and video go out of sync. This just happens when you use a compositor. There is always some lag. :/

**Comment by yshui** [comment]
Does #641 help with this?

**Comment by yshui** [comment]
Closed as being superseded by #641

**Pull request closed without merging**